tools/Artifacts.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    
+	<PropertyGroup>
+		<HiveArtifactsPath>$(MSBuildThisFileDirectory)..\artifacts\</HiveArtifactsPath>
```

Comment:
Does this always suffix with `/` or `\`?

Reply:
Yes, as is convention in MSBuild. Anything referring to a directory always has a trailing slash.